src/restconfig/src/main/java/org/geoserver/rest/resources/ResourceFinder.java

```diff
@@ -0,0 +1,49 @@
+/* (c) 2016 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.rest.resources;
+
+import org.geoserver.platform.resource.ResourceStore;
+import org.geoserver.rest.util.RESTUtils;
+import org.restlet.Finder;
+import org.restlet.data.Request;
+import org.restlet.data.Response;
+import org.restlet.resource.Resource;
+
+/**
+ *
```

Comment:
Please insert a comment here. For example, is this a Spring bean for handling REST requests for resources?